haiku-misc/gadgeteer/additional-files/gadgeteer.rdef.in

```diff
@@ -0,0 +1,16 @@
+
+resource app_signature "application/x-vnd.pjg-gadgeteer";
+
+resource app_version {
+	major  = 0,
+	middle = 8,
+	minor  = 1,
```

Comment:
Hmm, you need some extra steps to get the rdef file in the build directory.
I see 2 scenari:

#### A) Adding placeholders in `gadgeteer.rdef.in`:
```diff
 resource app_version {
-	major  = 0,
+	major  = @MAJOR@,
-	middle = 8,
+	middle = @MIDDLE@,
-	minor  = 1,
+	minor  = @MINOR@,
```
and a sed command:
```diff
 BUILD()
 {
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/gadgeteer.rdef.in \
+			> gadgeteer.rdef
+
 	make OBJ_DIR=objects
 }
```

#### B) Renaming `gadgeteer.rdef.in` to `gadgeteer.rdef` and using cp:
```diff
 BUILD()
 {
+	cp "$portDir"/additional-files/gadgeteer.rdef .
 	make OBJ_DIR=objects
 }
```

BTW, these steps are generally done in INSTALL but in this case we need them in BUILD because the rdef file is required right after the compilation.

Some may argue that we don't need placeholders because upstream might never release a new version.
So feel free to choose whatever option you think is best.